### ISSUE TITLE
NuGet 3.5.0 beta2 can't parse Microsoft.CodeAnalysis.Common 1.3.0

### ISSUE THREAD

**Issue by AndriySvyryd** [NuGet/Home#3010]
- Create .NET Core library project
- Update NETStandard.Library to 1.6.0
- Open NuGet Package Manager Installed tab

```
The schema version of 'NETStandard.Library' is incompatible with version 2.12.0.815 of NuGet. Please upgrade NuGet to the latest version from http://go.microsoft.com/fwlink/?LinkId=213942.
```

Using VS 2015 U3 25420 with NuGet 3.5.0 beta2 1484

**Comment by rohit21agrawal** [comment]
@AndriySvyryd this was a known issue and packages were published to myget with <serviceable> element and an updated xml namespace the day the feature was added. It was later decided that the namespace wouldn't be updated in the packages nuspec file. Not sure how you are running into those packages ,you might need to download the latest version of the packages?

CC : @rrelyea

**Comment by AndriySvyryd** [comment]
I am getting this from https://dotnet.myget.org/F/aspnet1/api/v3/index.json 
This is the feed we are using for RTM verification

**Comment by AndriySvyryd** [comment]
@muratg @Eilon We might need to update the feed / build

**Comment by AndriySvyryd** [comment]
Cleaning the caches again resolved the issue

**Comment by AndriySvyryd** [comment]
Now I get this with `Microsoft.CodeAnalysis.Common` and `Microsoft.CodeAnalysis.CSharp` 1.3.0

**Comment by Eilon** [comment]
Do we suspect the problem is with the package itself, the feed, or NuGet's UI?

**Comment by AndriySvyryd** [comment]
From @rohit21agrawal response it sounds like a package issue. Don't know whether it has been found/fixed for these particular packages though.

**Comment by rohit21agrawal** [comment]
@Eilon package itself. One way to verify is to check the xml namespace in the nuspec file of the package. I suspect it will be something ending with 2016/06 .

**Comment by AndriySvyryd** [comment]
Yep `<package xmlns="http://schemas.microsoft.com/packaging/2016/06/nuspec.xsd">`

**Comment by Eilon** [comment]
Ok. Do we suspect it's in the original package, or does uploading it to the feed mess something up?

**Comment by Eilon** [comment]
@AndriySvyryd I sent you contact info for the Roslyn folks.

cc @jaredpar @MattGertz here as well.

**Comment by jaredpar** [comment]
Why are we just seeing this error now?  Shouldn't the validation pass have caught this?

**Comment by rrelyea** [comment]
@AndriySvyryd - I think that there is a dotnet myget feed that replaced the aspnet myget feed. Are you using the right feed for verification?

**Comment by AndriySvyryd** [comment]
I am doing manual validation. The error only shows up when using the NuGet GUI. restoring / building the project works fine

**Comment by MattGertz** [comment]
RIght, it should be dotnet.myget.org IIRC.  Can you verify that's what you are using?

**Comment by AndriySvyryd** [comment]
@rrelyea @MattGertz I am using https://dotnet.myget.org/F/aspnet1/api/v3/index.json as per latest instructions

**Comment by MattGertz** [comment]
Well, hmm.  @jaredpar, we'd better take a look.

**Comment by AndriySvyryd** [comment]
Also it's not limited to the packages mentioned above. All Roslyn packages on that feed have this issue.

**Comment by jaredpar** [comment]
I was able to install these packages on my machine no issue.  I'm using VS 2015 Update 2.  Is this a bug introduced in Update 3 possibly?

**Comment by rohit21agrawal** [comment]
@AndriySvyryd could you clear all your caches and try once?

Clearing NuGet HTTP cache: C:\Users<username>\AppData\Local\NuGet\v3-cache
Clearing NuGet cache: C:\Users<username>\AppData\Local\NuGet\Cache
Clearing NuGet global packages cache: C:\Users<username>.nuget\packages\

**Comment by AndriySvyryd** [comment]
@rohit21agrawal  I did it 3 times already, doesn't help

**Comment by rrelyea** [comment]
@rohit21agrawal is going to go walk all the .nupkgs in the verification feed to see who has the "bad xmlns uri" that our first "pack w/ serviceable" created. #3012

**Comment by MattGertz** [comment]
Related to that, adding @cston to verify the version of NuGet used.

**Comment by cston** [comment]
We used `NuGet Version: 3.5.0.1437` to create the `Microsoft.CodeAnalysis.*` packages.

**Comment by jaredpar** [comment]
Where would this namespace be coming from?  I did a spot check of our nuspec files and they all have 2011 versions

``` xml
<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```

Here is the nuspec for [Microsoft.CodeAnalysis.Common](https://github.com/dotnet/roslyn/blob/stabilization/src/NuGet/Microsoft.CodeAnalysis.Common.nuspec) which is currently the title of the bug.

**Comment by cston** [comment]
@jaredpar The `xmlns` may be generated by `nuget.exe pack`.

**Comment by rohit21agrawal** [comment]
@jaredpar is this nuspec from the copy of the package uploaded on the server?

**Comment by cston** [comment]
The packages on the server have `<package xmlns="http://schemas.microsoft.com/packaging/2016/06/nuspec.xsd">`.

**Comment by jaredpar** [comment]
@rohit21agrawal yes

@cston so NuGet.exe overwrote the namespace we added in our nuspec?  I just double checked and we don't have that entry in our nuspec file.

**Comment by rohit21agrawal** [comment]
@cston that nuget version does add the new xmlns to the nuspec. We fixed it immediately after, you should use 3.5.0-beta2-1484 to pack

**Comment by rrelyea** [comment]
This is tracked by a Roslyn issue and being handled offline. Closing our issue.
